src/EditTableDialog.ui

```diff
@@ -429,6 +429,11 @@
             <string>Name</string>
            </property>
           </column>
+          <column>
+           <property name="text">
+            <string>Reference</string>
```

Comment:
I'd suggest renaming this to Details or Settings or something more generic. This way the column is not only useful for foreign keys but can also be used for check constraints etc.